pubspec.yaml

```diff
@@ -14,8 +14,8 @@ dependencies:
     sdk: flutter
 
   ## LikeMinds Chat SDK
-  likeminds_chat_fl: 1.1.0
-     #path: ../LikeMinds-Flutter-GroupChat-SDK
+  likeminds_chat_fl: 1.1.1
```

Comment:
Don't build like this while testing, otherwise fine for now

Reply:
okayy